docs/index.md

```diff
@@ -1,31 +1,23 @@
 Zanata Server
 =============
+Please visit [our about page](http://zanata.org/about/)
 
-TBD
-
-Features
---------
```

Comment:
This is probably still a good section to have. We should leave an xml comment reminding us to add it, at least.